cmd/simulacra/simulacra.go

```diff
@@ -204,9 +217,61 @@ func getConfigFromYaml(configPath string) (*Config, error) {
 	return &config, nil
 }
 
+// Parse the metadata image name with format 'projects/debian-cloud/global/images/debian-11-bullseye-v20230711'
+// and return the scope and image name.
+func parseMetadataImage(name string) (string, string, string, error) {
+	components := strings.Split(name, "/")
+	if len(components) < 5 {
+		return "", "", "", fmt.Errorf("image name from metadata must be of format 'projects/debian-cloud/global/images/debian-11-bullseye-v20230711' ")
```

Comment:
nit: if you aren't formatting anything, you can use `errors.New` instead of `fmt.Errorf`.

Reply:
```suggestion
		return "", "", "", errors.New("image name from metadata must be of format 'projects/debian-cloud/global/images/debian-11-bullseye-v20230711'")
```